packages/react/src/themes/teams/components/Icon/fontAwesomeIconStyles.ts

```diff
@@ -11,7 +11,7 @@ const brand = (content: string) => fontIcon(content, '"Font Awesome 5 Brands"')
 // Originally generated from:
 // https://github.com/Semantic-Org/Semantic-UI-CSS/blob/master/components/icon.css
 // Corrections were made to duplicate icon names and incorrectly mapped alternates
-const fontAwesomeIcons: ThemeIcons = {
+const fontAwesomeIcons: { [key: string]: ThemeIconSpec } = {
```

Comment:
may we use `ThemeIcons` type here as well?

Reply:
Yep, changed